### ISSUE TITLE
D3 bar chart tooltip test not passing

### ISSUE THREAD

**Issue by numoonchld** [freeCodeCamp/testable-projects-fcc#607]
#### Issue Description

Tooltip appears on mouseover on bars of the bar chart, but the two tooltip tests fail.

It works in the example though - but the tooltip in the example https://codepen.io/freeCodeCamp/full/GrZVaM looks very different compared to the tooltip in the curriculum tooltip challenge: https://learn.freecodecamp.org/data-visualization/data-visualization-with-d3/add-a-tooltip-to-a-d3-element/

#### Browser Information

* Browser Name, Version: Chrome Version 69.0.3497.100 (Official Build)
* Operating System: high sierra 10.13.6
* Mobile, Desktop, or Tablet: notebook

#### Your Code / Link to Your Pen
https://codepen.io/numoonchld/pen/YOZzeG?editors=0010

#### Screenshot

![image](https://user-images.githubusercontent.com/17598505/45696041-4ef9c580-bb80-11e8-851f-9ac40e03ebb0.png)

**Comment by tbushman** [comment]
Project uses external tooltip library https://popper.js.org/  
  
@numoonchld Thanks for reporting. There was a fix that may fix this and it hasn't been merged to the CDN yet. I'll check back with you.

**Comment by tbushman** [comment]
@numoonchld My guess is that the delay on the tooltip visibility causes the test to fail. Are you using tooltip.js or popper.js ?  
If tooltip.js, there's a delay option. Can you please try adding the `options.delay` properties as described in https://popper.js.org/tooltip-documentation.html and see if that fixes it? I don't see a similar option for popper.js ...

**Comment by numoonchld** [comment]
I've loaded the popper.js CDN as it is a dependency of bootstrap I believe. 

I don't think I'm using either popper.js or tooltip.js for the tooltip, because the challenge here: https://learn.freecodecamp.org/data-visualization/data-visualization-with-d3/add-a-tooltip-to-a-d3-element/ doesn't say anything about using popper or tooltip. 

I do a .append('title) and set the tooltip text using .text("tooltip-text"). I added an "id" and "data-gdp" property using .attr()

Also, the tooltip that appears after solving the challenge has a delay similar to the delay in the linked codepen project.

I understand that popper can be used to make a tooltip - but here I'm just trying to pass the test suite. The challenge passes with .append("title") and .text() but this test suite for codepen doesn't. 

Will only using popper get the tests to pass?
